packages/css/src/accordion/accordion.scss

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+@import "../../utils/breakpoint";
+
+.amsterdam-accordion__header {
+  display: flex;
+  margin-block: 0;
+  margin-inline: 0;
+}
+
+.amsterdam-accordion__button {
+  background-color: transparent;
+  border: 0;
+  color: var(--amsterdam-accordion-button-color);
+  cursor: pointer;
+  display: flex;
+  font-family: var(--amsterdam-accordion-button-font-family);
+  font-size: var(--amsterdam-accordion-button-font-size-narrow);
+  font-weight: var(--amsterdam-accordion-button-font-weight);
+  justify-content: space-between;
+  line-height: var(--amsterdam-accordion-button-line-height);
+  padding-block: 0.75rem;
+  padding-inline: 1rem;
+  width: 100%;
+
+  @media screen and (width > $amsterdam-breakpoint) {
+    font-size: var(--amsterdam-accordion-button-font-size-wide);
+  }
+
+  &:focus {
+    outline-offset: var(--amsterdam-accordion-button-focus-outline-offset);
```

Comment:
Not sure what was agreed on focus outline but a blue border with radius (in Chrome) looks weird around a grey border without radius.

Reply:
Agreed, I'll remove the `focus-visible` style